Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cv"
-version = "0.2.2"
+version = "0.2.1"
```

Comment:
Why reverting? I have a strong feeling that every code change have to change a version.